dashboard/client/dashboard.js

```diff
@@ -5,8 +5,74 @@ Template.dashboard.onCreated(function () {
   // Get reference to template instance
   const templateInstance = this;
 
-  // Create reactive variable for Elasticsearch host
+  // Create reactive variable for Elasticsearch host & data
   templateInstance.elasticsearchHost = new ReactiveVar();
+  templateInstance.elasticsearchData = new ReactiveVar();
+
+  const queryParams = {
+    size: 0,
+    body: {
+      query: {
+        filtered: {
+          query: {
+            bool: {
+              should: [
+                {
+                  wildcard: {
+                    request_path: {
+                      // Add '*' to partially match the url
+                      value: '/*',
+                    },
+                  },
+                },
+              ],
+            },
+          },
+        },
+      },
+      aggs: {
+        // Create data range with interval
+        requests_over_time: {
+          date_histogram: {
+            field: 'request_at',
+            interval: 'month',
+          },
+          aggs: {
+            // Get average value of response time for each range
+            avg_response_time: {
+              avg: {
```

Comment:
> Show average response times (from API Umbrella) for the selected API backend

@brylie I have clearly the task show **average**, no median, no percentiles, no skewness or so on. Just average. And I'm using [Average metric from Elasticsearch document](https://www.elastic.co/guide/en/elasticsearch/reference/2.3/search-aggregations-metrics-avg-aggregation.html). If you don't agree with someone then open the following issue where you will explain your vision.